### ISSUE TITLE
including a form creates duplicate CSS-IDs

### ISSUE THREAD

**Issue by fritzmg** [contao/core-bundle#956]
If you give a form a CSS-ID and include the form in the front end, the CSS-ID will be used for _both_ the __container element__ and the __form itself__.
### Reproduction

1. Create a new form in the form generator and give it a CSS-ID, e.g. `foo`.
2. Include the form in any way in the front end:
    * Via an include content element.
    * Via `<?= \Controller::getForm(…) ?>`
    * Via `{{insert_form::*}}`.
3. Analyze the HTML output. It will be the following in all cases (either with `ce_form` or `mod_form`):
```html
<div class="ce_form block" id="foo">
  <form action="foo.html" id="foo" method="post" enctype="application/x-www-form-urlencoded">
    …
  </form>
</div>
```

**Comment by fenepedia** [comment]
Kann den Fehler bestätigen und reproduzieren.

**Comment by leofeyer** [comment]
Fixed in ee9ba7f7d7795f84b1202d337634b85463b5436c.
